src/popup/components/TransactionLabel.vue

```diff
@@ -230,7 +229,7 @@ export default defineComponent({
 
       const translation = transactionListTypes[txType.value!] || transactionTypes[txType.value!];
 
-      if (txType.value && txType?.value?.includes('name')) {
+      if (txType.value && txType.value?.includes('name')) {
```

Comment:
Wasn't that fixed in your other PR? If not - why `txType.value` is doubled here?

Reply:
must be this way, because txType.value sometimes can be "false"

Reply:
yes and that's solved by the question mark after it

Reply:
![image](https://user-images.githubusercontent.com/9008074/223072263-8c453485-ca31-4b6e-9ec0-84422129499d.png)

Reply:
Ah yes I missed that it would have wrong type, never mind then